### DISCUSSION TITLE
Laravel creates a soft backup of ENV files.

### DISCUSSION THREAD

**Discussion by bhargavraviya** [laravel/framework#48051] [Ideas]
.env file backup: soft means command backup of all env keys in one file without values, like in the env.example. It is helping to save a file with a new key and value for Stripe and other configurations for env secure save. Never forget to define variables in production. also update in git because, in the future, I forgot some variables in env at that time because of a problem finding code for env variable use.

And

make command for the env variable, because some developers use the env variable to direct code, but it is now the proper way because of the caching issue at that time. At that time, create env and code between in the config file, like STRIPE_KEY, then variable also in the config file stripe_key => env('STRIPE_KEY','')
```
php artisan backup:env-soft
```

**Comment by henzeb** [comment]
There is env:encrypt. Would that do what you want? Because I honestly don't understand what you are trying to achieve.

**Reply by bhargavraviya** [reply]
Example
.env file 
```
DB_HOST=127.0.0.1
```
soft backup like this 
.env.example file 
```
DB_HOST=
```

only with value to store .env.example. it is helping to save key of env file
